C/src/modal_models/FurutaPendulum/README.md

```diff
@@ -0,0 +1,44 @@
+# Furuta Pendulum
+
+A [Furuta pendulum](https://en.wikipedia.org/wiki/Furuta_pendulum) is a classic control system problem often used to teach feedback control. 
+It consists of a vertical shaft driven by motor, a fixed arm extending out at 90 degrees from the top of the shaft, and a pendulum at the end of the arm.
+The goal is to rotate the shaft to impart enough energy to the pendulum that it swings up, to then catch the pendulum and balance it so that the pendulum remains above the arm.
+Each of these steps requires a different control behavior, which makes a controller a prime candidate for a modal model.
+
+The example here replicates [a solution by Eker et al.](https://ptolemy.berkeley.edu/papers/02/IFAC/IFAC.pdf)
+It uses a simple forward-Euler simulation of an actual pendulum and the control logic provided by Eker et al. to implement a modal controller in Lingua Franca.
+
+The program is configured to automatically plot the results of the simulation.
+
+
+## Prerequisites
+
+The executable examples use a custom build script that compiles the program, as usual, but also executes the program and processes its output to generate and open a plot. 
+It relies on [cmake](https://cmake.org/) and [gnuplot](http://www.gnuplot.info/) being available on the PATH.
+The script is not compatible to Windows systems.
```

Comment:
Is this true? There are Windows-specific checks in there though...

Reply:
Oh yes, this might be deprecated but I still need to confirm that the script work on Windows. I will try to do that today.

Reply:
I tested the script on Windows with MinGW and removed the note.